export_parse.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+import argparse
+import pandas as pd
+from os import makedirs, getcwd
+from os.path import exists, join
+from shutil import copy
+import sys
+
+def export_parse(input_file, source_path, out_path, failed):
+    """
+    Takes in an export csv from cellswiper, filter out either the passing or failing annotations,
+    and copies those files to the out_path
+    """
+    try:
+        annotations = pd.read_csv(input_file)
+    except:
+        print("Could not read input file")
+        sys.exit(1)
+    
+    if not exists(source_path):

```

Comment:
`source_path` is the parent folder of where the images are kept. So if you wanted to copy images from /data/cellswiper/cat_batch, you could run the script from /data/cellswiper and pass in cat_batch as the source_folder